packages/rocketchat-api/server/helpers/getUserInfo.js

```diff
@@ -56,5 +56,9 @@ RocketChat.API.helperMethods.set('getUserInfo', function _getUserInfo(user) {
 		preferences: getUserPreferences(),
 	};
 
+	me.telephoneNumber = '';
+	if (('services' in user) && ('viasatsso' in user.services) && ('telephoneNumber' in user.services.viasatsso)) {
+		me.telephoneNumber = user.services.viasatsso.telephoneNumber;
+	}
```

Comment:
@kb0304 How hard would it be here to use the configuration value `Contacts_Phone_Custom_Field_Name` from the contacts configuration page?  I think that might be an easy way to avoid hard coding here?  Can you implement that instead?  Thanks......

Reply:
Done